### PR TITLE
Allow to disable js/css assets auto-reload in system debug plugin

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/16381.

### Summary of Changes

Adds an option to the debug system plugin to allow to disable the js/css assets auto-reload on every page load in system debug plugin.

### Testing Instructions

1. Apply patch
2. Enable debug system plugin and debug in global config
3. Check there is a media version hash in assets (js/css) in the page source code
3. Now, in the debug system plugin disable the new "Refresh Assets" option
4. Save, reload the page and check there is no media version hahs in assets (js/css) in the page source code

### Expected result

Allow to disable js/css assets auto-reload in debug system plugin.

### Actual result

Not allowed to disable js/css assets auto-reload in debug system plugin.

### Documentation Changes Required

none.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_system_debug.ini [hunk 1]
```diff
@@ -43,8 +43,8 @@ PLG_DEBUG_FIELD_LOG_PRIORITIES_INFO="Info"
 PLG_DEBUG_FIELD_LOG_PRIORITIES_LABEL="Log Priorities"
 PLG_DEBUG_FIELD_LOG_PRIORITIES_NOTICE="Notice"
 PLG_DEBUG_FIELD_LOG_PRIORITIES_WARNING="Warning"
-PLG_DEBUG_FIELD_REFRESH_MEDIA_VERSION_LABEL="Refresh assets"
-PLG_DEBUG_FIELD_REFRESH_MEDIA_VERSION_DESC="If enabled will, on each page reload, add a different hash to every script/stylesheets files with auto version so they never use the browser cache."
+PLG_DEBUG_FIELD_REFRESH_ASSETS_LABEL="Refresh assets"
+PLG_DEBUG_FIELD_REFRESH_ASSETS_DESC="If enabled will, on each page reload, add a different hash to every script/stylesheets files with auto version so they never use the browser cache."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_debug.ini]: Swap lines to be in alphabetical order.

Suggestions:  
`If enabled, on each page reload, will add`
`stylesheets` to `stylesheet`

**Comment by ggppdk** [comment]
I would say that this works,
-- and it would quite useful for developers
-- and for running the site faster when debug is ON

But it may cause problems due to misunderstanding / misuse of it by end users ! 
If some end-users, use the new option to disable the adding of the Version URL variable to the JS / CSS files

-- then **when at some point they re-enable debug after a joomla upgrade, then their browsers maybe use old cached data** from a previous Joomla version
(at least since it is inside a plugin they will probably never find it)

The description of new parameter says "so they never use the browser cache",

and "no browser cache" will sound bad to end users and they will disable it
also even for people that understand it , they may disable it , and then just leave it off and forget about it

maybe a better description (language string) is needed here

**Comment by brianteeman** [comment]
The language strings make sense to me

**Comment by andrepereiradasilva** [comment]
@Quy @brianteeman english is not my native language, please feel free ot change the language strings to what you think is more correct.

> But it may cause problems due to misunderstanding / misuse of it by end users !
If some end-users, use the new option to disable the adding of the Version URL variable to the JS / CSS files
> -- then when at some point they re-enable debug after a joomla upgrade, then their browsers maybe use old cached data from a previous Joomla version
> (at least since it is inside a plugin they will probably never find it)

@ggppdk not sure i understand, you are describing a hypothetical scenario in which a end-user deliberate enabled the debug plugin and deliberate disabled this new option in a live production site? ... if so is your opinion that this something joomla core need to that in consideration?

**Comment by ggppdk** [comment]
@andrepereiradasilva
This PR is good, what i described will be rare

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on a511e614c5fda880d945e86783e6b61fa089b2ed<br /><br />Working well. @GeraintEdwards Can you test when you get a minute please.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16406">issues.joomla.org/tracker/joomla-cms/16406</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a058bf9b09b9b6eac680842bb07ea378ff949a3f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16406">issues.joomla.org/tracker/joomla-cms/16406</a>.</sub>

**Comment by tonypartridge** [comment]
Can someone run drone please

**Comment by andrepereiradasilva** [comment]
just updated branch to rerun tests

**Comment by tonypartridge** [comment]
Great, can someone put an RTC on this? since it has passed two tests? @franz-wohlkoenig ?

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by tonypartridge** [comment]
My Hero!

**Comment by Quy** [comment]
@andrepereiradasilva 

> Enable debug system plugin and debug in global config

It is not necessary to enable debug in global config for this to work. Is it ok?

**Comment by andrepereiradasilva** [comment]
@Quy fixed just added JDEBUG condition (forgot it)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1036aaf6bec9e03d9ddeb69b9e743f5584ff0c41<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16406">issues.joomla.org/tracker/joomla-cms/16406</a>.</sub>

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on 1036aaf6bec9e03d9ddeb69b9e743f5584ff0c41<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16406">issues.joomla.org/tracker/joomla-cms/16406</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**